service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/azure/database/AzureDatabaseUtilsRunner.java

```diff
@@ -150,7 +152,10 @@ public void pgDumpDatabase(
             new V1EnvVar().name(PARAM_BLOB_CONTAINER_NAME).value(blobContainerName),
             new V1EnvVar()
                 .name(PARAM_BLOB_CONTAINER_URL_AUTHENTICATED)
-                .value(blobContainerUrlAuthenticated));
+                .value(blobContainerUrlAuthenticated),
+            new V1EnvVar().name(PARAM_ENCRYPTION_KEY).value(encryptionKey)
```

Comment:
are we going to make this a k8s secret in the future?

Reply:
once this gets merged into our feature branch, I'll update my followup PR to pass this in as a secret